Marlin/Configuration.h

```diff
@@ -1595,7 +1595,7 @@
 // Delay (in milliseconds) before the next move will start, to give the servo time to reach its target angle.
 // 300ms is a good value but you can try less delay.
 // If the servo can't reach the requested position, increase it.
-#define SERVO_DELAY 300
+#define SERVO_DELAY { 300 }
```

Comment:
This needs to be reproduced in all other `Configuration.h` files in `example_configurations`.